plugin/src/main/java/app/cash/paraphrase/plugin/ArgumentTypeResolver.kt

```diff
@@ -109,11 +109,11 @@ private val compatibleTokenTypes: Map<TokenType, List<TokenType>> = mapOf(
   DateTimeWithZone to emptyList(),
   Offset to listOf(TimeWithOffset, DateTimeWithOffset, DateTimeWithZone),
   SpellOut to listOf(Choice, Number, Ordinal, Plural, SelectOrdinal),
-  Ordinal to listOf(Choice, Plural, SelectOrdinal),
+  Ordinal to listOf(Choice, SelectOrdinal),
   Duration to emptyList(),
-  Choice to listOf(Ordinal, Plural, SelectOrdinal),
-  Plural to listOf(Choice, Ordinal, SelectOrdinal),
+  Choice to listOf(Ordinal, SelectOrdinal),
+  Plural to listOf(Choice, Number, Ordinal, Plural, SelectOrdinal, SpellOut),
```

Comment:
Is it intentional that `Plural` is included in its own list?

Reply:
Nope, good catch!